lib/src/screens/send/send_page.dart

```diff
@@ -386,16 +386,14 @@ class SendPage extends BasePage {
                     amount: S.of(context).send_amount,
                     amountValue:
                         sendViewModel.pendingTransaction!.amountFormatted,
-                    fiatAmountValue:
-                        sendViewModel.pendingTransactionFiatAmount +
-                            ' ' +
-                            sendViewModel.fiat.title,
+                    fiatAmountValue: sendViewModel.balanceViewModel.disableFiat
+                        ? '' : sendViewModel.pendingTransactionFiatAmount +
+                            ' ' + sendViewModel.fiat.title,
                     fee: S.of(context).send_fee,
                     feeValue: sendViewModel.pendingTransaction!.feeFormatted,
-                    feeFiatAmount:
-                        sendViewModel.pendingTransactionFeeFiatAmount +
-                            ' ' +
-                            sendViewModel.fiat.title,
+                    feeFiatAmount: sendViewModel.balanceViewModel.disableFiat
+                        ? '' : sendViewModel.pendingTransactionFeeFiatAmount +
+                            ' ' + sendViewModel.fiat.title,
```

Comment:
Better to create new getter like `pendingTransactionFiatAmountFormatted` which will return pendingTransactionFiatAmount + ' ' +  fiat.title and if fiat api is disabled then return empty line.
Same for `pendingTransactionFeeFiatAmount` for send view model.